tests/python/unittest/test_ndarray.py

```diff
@@ -1653,6 +1653,14 @@ def test_ndarray_nan_comparison():
     for i in (np.isnan(data1_grad))[1][0].flatten():
         assert i == True
 
+
+def test_zero_from_numpy():
```

Comment:
Can you add more test cases including
1. scalar tensors.
2. zero-size tensors.
3. `zero_copy=False`.
4. Perturbing the elements of numpy ndarray will equivalently affect mxnet NDArray, and vice versa.
5. Deleting either one of them should not obliterate the content of the other one when `zero_copy=True` and `False`.

Reply:
Done